packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -227,11 +227,11 @@ impl<'m> Embedded<'m> {
         Ok(Self {
             hash_stdlib,
             hash_util,
-            batches_prototype,
-            experiment_ctx_prototype,
-            sim_init_ctx_prototype,
-            gen_ctx,
-            gen_state,
+            _batches_prototype: batches_prototype,
```

Comment:
@litvand Do we need all of these still?

Reply:
No, I didn't notice that they aren't used outside of this function, where they are available as local variables anyways.